### ISSUE TITLE
"Error mounting /dev/sda" and sh1mmer not making any changes on hana (100e 2G MTK)

### ISSUE THREAD

**Issue by alectrona6400** [CoolElectronics/sh1mmer.me#38]
Hello, I am trying to deprovision an old chromebook (Lenovo 100e 2nd gen MTK) that was decommissioned which had the enrollment left on it. Tried contacting the original district about it and they just told me to send it back so they could recycle it, which is kinda stupid considering they were the ones who let it go. I want to fix this rather than killing it off, so I looked into the sh1mmer exploit. While I was able to get the image to load, I could not get it to actually make changes to the system. Also, when I was building it with wax4web, it says a few things:
```
ERROR: mounting /dev/sda at /tmp/tmp.O5BV2M failed:
mount: mounting /dev/sda on /tmp/tmp.O5BV2M failed: Invalid argument
ERROR: mounting /dev/sda at /tmp/tmp.LZWGyS failed:
mount: mounting /dev/sda on /tmp/tmp.LZWGyS failed: Invalid argument
```
I suspect it was not entirely successful with the operation? I couldn't see any errors on the chromebook but I tried doing the following in order:
-De-provision
-disable block_devmode
-enable USB booting

and it still won't let me into devmode, which also leads me to think that it didn't deprovision the device and enable USB booting either. This device clearly has not been updated to 111 considering the bootscreen is the old style one and not the newer one, so it should work in theory. Is there a specific version I need to download and recover to before I can get this working correctly? I'm not sure what version of Chrome OS it is on but I know it was last used in February 2022. Also, I tried building a legacy shim from my own linux VM and that did not work.

**Comment by amelia-bartlett** [comment]
try contacting Google directly?  They might deprovision it for you if you can provide proof of ownership.

**Comment by ghost** [comment]
Hana board? Did you use the legacy  sh1m builder?
EDIT: Sorry, I wasn't paying attention and didn't see the part where you used Linux VM. What VM did you use?
I do not believe this is a wax or Sh1mmer issue, and most likely an error with the tools you are using.

Some things to try:

reinstalling the shim/making sure you installed the correct shim (The wax4web error could've been you didn't use the legacy wax4web [https://sh1mmer.me/buildlegacy] or an installation error)
using an actual linux distro when building, as some distros use different commands
using the kiosk v80 deprovision method (coolelectronics.me/unroll)
downgrading (chrome100.dev)
switching to another device to build on
What the comment above me said about going to google

I hope this helps!

**Comment by amelia-bartlett** [comment]
> Hello, I am trying to deprovision an old chromebook (Lenovo 100e 2nd gen MTK) that was decommissioned which had the enrollment left on it. Tried contacting the original district about it and they just told me to send it back so they could recycle it, which is kinda stupid considering they were the ones who let it go. I want to fix this rather than killing it off, so I looked into the sh1mmer exploit. While I was able to get the image to load, I could not get it to actually make changes to the system. Also, when I was building it with wax4web, it says a few things:
> 
> ```
> ERROR: mounting /dev/sda at /tmp/tmp.O5BV2M failed:
> mount: mounting /dev/sda on /tmp/tmp.O5BV2M failed: Invalid argument
> ERROR: mounting /dev/sda at /tmp/tmp.LZWGyS failed:
> mount: mounting /dev/sda on /tmp/tmp.LZWGyS failed: Invalid argument
> ```
> 
> I suspect it was not entirely successful with the operation? I couldn't see any errors on the chromebook but I tried doing the following in order: -De-provision -disable block_devmode -enable USB booting
> 
> and it still won't let me into devmode, which also leads me to think that it didn't deprovision the device and enable USB booting either. This device clearly has not been updated to 111 considering the bootscreen is the old style one and not the newer one, so it should work in theory. Is there a specific version I need to download and recover to before I can get this working correctly? I'm not sure what version of Chrome OS it is on but I know it was last used in February 2022. Also, I tried building a legacy shim from my own linux VM and that did not work.

Do you know what board it is?  If it is one of a few (think its `hana` and `coral`) you need to build a legacy shim for it to even boot (legacy shims have no graphics, and are built on top of shim images that are so old that they dont have image display support).  If you can get a beautifulworld shim to boot but not work, try building a legacy one anyway (they still work)

The errors you are getting with wax4web sound like potentially an issue at their end.  Did you try building locally?  (I've personally never used wax4web - I cloned the repo locally and built it on my Linux machine since im not waiting for some Javascript or WebAssembly or whatever tf it is vm to build something when I have a Ryzen 5 5000 series)

I have also only ever tried against a `volteer` board, so idk maybe its a board-specific issue.

**Comment by alectrona6400** [comment]
I did use the legacy builder, both on web and local. I actually managed to get it working by restoring to Chrome OS 107 and then doing it again for some reason. However it won't work when I go and do it without restoring to 107. Not sure what could be causing this but there is an issue with the legacy builder on certain lenovo systems where it... doesn't seem to do anything even if I can get into the menu and run commands?
